caluma-demo/app/components/navbar.js

```diff
@@ -0,0 +1,3 @@
+import Component from "@ember/component";
+
+export default class NavbarComponent extends Component {}
```

Comment:
If you add `"template-only-glimmer-components": true` to `config/optional-features.js` you can delete this file :smile:

Reply:
Nice, thanks!